contracts/FragToken.sol

```diff
@@ -12,9 +12,16 @@ contract FRAGToken is ERC20, ERC20Permit, Ownable{
     uint256 constant INITIAL_SUPPLY = 10_000_000_000 * (10**DECIMALS); 
     uint256 private constant _TIMELOCK = 1 weeks;
 
-    mapping(address => uint256) private _locksAmount;
-    mapping(address => uint256) private _locksBlock;
-    mapping(address => uint256) private _locktime;
+    struct LockInfo {
+        address sender;
+        uint256 locktime;
+        uint256 amount;
+        bool unlocked;
+    }
+
+    LockInfo private _lockInfo;
```

Comment:
why this? this will trigger more gas usage as it writes on state.

Reply:
not needed at all, you're right. Removed.